tools/sync-docs.py

```diff
@@ -24,12 +24,17 @@
 
         var items = [];
         $.each( data, function(_, version) {
+            if (version == data[1]) {
+                latest = " (latest)";
```

Comment:
Hmm, maybe this should be "(latest stable)"? Because "devel" is obviously later, so this is confusing.